GameServerLib/GameObjects/GlobalData.cs

```diff
@@ -0,0 +1,20 @@
+﻿using GameServerCore.Domain;
+
+namespace GameServerLib.GameObjects.GlobalData
```

Comment:
Move this file to `GameServerLib/Content` and change namespace.
```suggestion
namespace LeagueSandbox.GameServer.Content
```